qt/scientific_interfaces/Muon/ALCPeakFittingModel.cpp

```diff
@@ -7,15 +7,14 @@
 #include "ALCPeakFittingModel.h"
 
 #include "MantidAPI/AlgorithmManager.h"
+#include "MantidAPI/AlgorithmRuntimeProps.h"
 #include "MantidAPI/CompositeFunction.h"
 #include "MantidAPI/ITableWorkspace.h"
 #include "MantidAPI/MatrixWorkspace.h"
 #include "MantidAPI/TableRow.h"
 #include "MantidAPI/TextAxis.h"
 #include "MantidAPI/WorkspaceFactory.h"
-
 #include <Poco/ActiveResult.h>
```

Comment:
I think this include is not necessary anymore after adding the algorithm subscriber?